data/aerial-photography/index.html

```diff
@@ -27,12 +28,12 @@ <h3 id="GOOGLE">
     </a>
     <ul class="dotless">
       <li>Natural Color (RGB)
-      <li>~6" pixels
+      <li>~6-inch pixels
     </ul>
-    <p>AGRC owns a public sector license to Google's statewide 6" aerial photography. Contractors and partners may also qualify. The Google license is for color aerial photography, typically collected within 3 years, from the spring, summer or fall. The imagery is statewide. The resolution is 6 inch or better with a horizontal positional accuracy to achieve or exceed one meter (C90) in most areas without significant vertical relief.</p>
+    <p>AGRC owns a public sector license to Google's statewide 6-inch aerial photography. Contractors and partners may also qualify. The Google license is for color aerial photography, typically collected within 3 years, from the spring, summer, or fall. The imagery is statewide, and the resolution is 6 inch or better with a horizontal positional accuracy to achieve or exceed 1 meter (C90) in most areas without significant vertical relief.</p>
   </div>
   <div class="grid__col grid__col--12-of-12 package-content">
-    <p><strong>This is a licensed product.</strong> To see if you are eligable for a license to use this imagery product, head over to the <a href="{{ '/discover/' | prepend: site.baseurl }}">Discover</a> section to get the most accurate information.</p>
+    <p><strong>This is a licensed product.</strong> To see whether you are eligible for a license to use the Google Imagery product, head over to our <a href="{{ '/discover/' | prepend: site.baseurl }}">Discover page</a> to get the most accurate information.</p>
```

Comment:
"Google Imagery product"--just one product? Or should this be "to use the Google Imagery products"?

Reply:
let's leave 'product' out all together

Reply:
and the rest looks good to change